.github/CODEOWNERS

```
@@ -56,6 +47,34 @@
 /Code/Tools/SceneAPI/ @o3de/sig-content-maintainers
 /Gems/SceneProcessing/ @o3de/sig-content-maintainers
 
+# SIG-Content Viewport ownership area
+/Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/FocusMode/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/Picking/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/ @o3de/sig-content-maintainers
+/Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ @o3de/sig-content-maintainers
+/Code/Framework/AzFramework/AzFramework/Viewport/ @o3de/sig-content-maintainers
+/Code/Framework/AzFramework/AzFramework/Visibility/ @o3de/sig-content-maintainers
+
+# SIG-Content White Box ownership area
+/AutomatedTesting/Gem/PythonTests/WhiteBox/ @o3de/sig-content-maintainers
+/Gems/WhiteBox/ @o3de/sig-content-maintainers
+
+# SIG-Simulation ownership area
+/Code/Framework/AzFramework/AzFramework/Physics @o3de/sig-simulation-maintainers
+/Gems/Blast/ @o3de/sig-simulation-reviewers @o3de/sig-simulation-maintainers
```

The first line is missing sig-simulation-reviewers
```suggestion
/Code/Framework/AzFramework/AzFramework/Physics @o3de/sig-simulation-reviewers @o3de/sig-simulation-maintainers
/Gems/Blast/ @o3de/sig-simulation-reviewers @o3de/sig-simulation-maintainers
```

---

Good spot, will fix :+1: